quicklogic/common/cmake/quicklogic_device.cmake

```diff
@@ -45,9 +45,14 @@ function(QUICKLOGIC_DEFINE_DEVICE_TYPE)
   set(VPR_DB_FILE "db_vpr.pickle")
   set(ARCH_XML "arch.xml")
 
+  set(DEVICE_DIR_DATA ${DEVICE})
+  if(${DEVICE} STREQUAL "ql-pp3")
+	  set(DEVICE_DIR_DATA "ql-pp3e")	# FIXME: use PP3 timing data when it will be available
```

Comment:
I suppose it would be better to use data from `eos-s3` as eg. routing timings there do not cause warning messages when converted to the routing model in VPR.

Reply:
The configuration is now switched to use EOS-S3 timing data.

Reply:
ok, thanks.